worldguard-legacy/src/main/java/com/sk89q/worldguard/bukkit/WorldGuardPlugin.java

```diff
@@ -129,6 +132,19 @@ public void onEnable() {
         configureLogger();
 
         getDataFolder().mkdirs(); // Need to create the plugins/WorldGuard folder
+        
+        // Load in custom flags
+        for (Plugin plugin : this.getServer().getPluginManager().getPlugins()) {
+            CustomFlagBroker broker = new CustomFlagBrokerImpl();
+            if (plugin instanceof CustomFlagProvider) {
+                CustomFlagProvider provider = (CustomFlagProvider)plugin;
+                try {
+                    provider.addCustomFlags(broker);
+                } catch (Exception e) {
+                    // Ignore
```

Comment:
You should at least print the exceptions.

Also, I could think of a safer way in the face of exceptions:
````diff
+                try {
+                    provider.clear(); // clear internal list
+                    provider.addCustomFlags(broker); // add flags to an internal list
+                    provider.commit(); // commit list to DefaultFlag.flagsList
+                } catch (Exception e) {
````